pyserini/util.py

```diff
@@ -21,7 +21,33 @@
 import tarfile
 from tqdm import tqdm
 from urllib.request import urlretrieve
-
+from collections import OrderedDict
+
+INDEX_MAPPING = OrderedDict(
+    [
+        (
+            'ms-marco-passage',
+            {
+                'urls': {'uwaterloo': 'https://git.uwaterloo.ca/jimmylin/anserini-indexes/raw/master/index-msmarco-passage-20191117-0ed488.tar.gz'},
```

Comment:
I use a dictionary here. We may choose the mirror used for download later, but now I use the first one by default.
I also add md5 of the index.

Reply:
sg